src/components/dialog-full-screen/dialog-full-screen.stories.mdx

```diff
@@ -660,6 +660,55 @@ to have a possibility to manage active `Tabs` group
   </Story>
 </Preview>
 
+### With help
+
+<Preview>
+  <Story name="with help" parameters={{ viewports: [500, 1400] }}>
```

Comment:
I don't think we need the viewports here, that was just for responsive stuff IIRC